components/brave_wallet_ui/components/extension/confirm-transaction-panel/confirm-zcash-transaction-panel.tsx

```diff
@@ -0,0 +1,185 @@
+  )
+
+  // custom hooks
+  const {
+    fromAccount,
+    fromOrb,
+    toOrb,
+    transactionDetails,
+    transactionsNetwork,
+    transactionTitle,
+    selectedPendingTransaction,
+    onConfirm,
+    onReject,
+    queueNextTransaction,
+    transactionQueueNumber,
+    transactionsQueueLength
+  } = usePendingTransactions()
+  const originInfo = selectedPendingTransaction?.originInfo ?? activeOrigin
+
+  // state
+  const [selectedTab, setSelectedTab] = React.useState<confirmPanelTabs>('transaction')
+
+  // methods
+  const onSelectTab = React.useCallback(
+    (tab: confirmPanelTabs) => () => setSelectedTab(tab),
+  [])
+
+  // render
+  if (
+    !transactionDetails ||
+    !selectedPendingTransaction ||
+    !transactionsNetwork ||
+    !fromAccount
+  ) {
+    return (
+      <StyledWrapper>
+        <Skeleton width={'100%'} height={'100%'} enableAnimation />
+      </StyledWrapper>
+    )
+  }
+
+  return (
+    <StyledWrapper>
+
+      <TopRow>
+        <NetworkText>{transactionsNetwork.chainName}</NetworkText>
+        <TransactionQueueSteps
+          queueNextTransaction={queueNextTransaction}
+          transactionQueueNumber={transactionQueueNumber}
+          transactionsQueueLength={transactionsQueueLength}
+        />
+      </TopRow>
+
+      <AccountCircleWrapper>
+        <FromCircle orb={fromOrb} />
+        <ToCircle orb={toOrb} />
+      </AccountCircleWrapper>
+      <URLText>
+        <CreateSiteOrigin
+          originSpec={originInfo.originSpec}
+          eTldPlusOne={originInfo.eTldPlusOne}
+        />
+      </URLText>
+      <FromToRow>
+        <Tooltip
+          text={fromAccount.name}
+          isAddress={true}
+          position='left'
+        >
+          <AccountNameText>{fromAccount.name}</AccountNameText>
+        </Tooltip>
+
+        {transactionDetails.recipient && transactionDetails.recipient !== fromAccount.address &&
+          <>
+            <ArrowIcon />
+            <Tooltip
+              text={transactionDetails.recipient}
+              isAddress={true}
+              position='right'
+            >
+              <AccountNameText>{transactionDetails.recipientLabel}</AccountNameText>
+            </Tooltip>
+          </>
+        }
+      </FromToRow>
+
+      <TransactionTypeText>{transactionTitle}</TransactionTypeText>
+      <TransactionAmountBig>
+        {new Amount(transactionDetails.valueExact)
+            .formatAsAsset(undefined, transactionDetails.symbol)
+        }
+      </TransactionAmountBig>
+      <TransactionFiatAmountBig>
+        {
+          new Amount(transactionDetails.fiatValue).formatAsFiat(defaultCurrencies.fiat)
+        }
+      </TransactionFiatAmountBig>
+
+      <TabRow>
+        <PanelTab
+          isSelected={selectedTab === 'transaction'}
+          onSubmit={onSelectTab('transaction')}
+          text='Transaction'
+        />
+        <PanelTab
+          isSelected={selectedTab === 'details'}
+          onSubmit={onSelectTab('details')}
```

Comment:
Can inline this action

```suggestion
          onSubmit={() => setSelectedTab('details')}
```

Reply:
This is copy of bitcoin transaction panel and will be customized in the future, i'd keep it unchanged to make it easier extracting common parts in the future.